compiler/passes/normalize.cpp

```diff
@@ -2693,13 +2712,12 @@ static void makeExportWrapper(FnSymbol* fn) {
     }
   }
 
-  // TODO: Also check if return type is an array.  Don't make two wrappers,
-  // pls.
-  if (argsToReplace) {
-    // We have at least one array argument.  Need to make a version of this
-    // function that can be exported
+  if (argsToReplace || returnsArray(fn)) {
+    // Either we have at least one array argument, or we return an array.
+    // Need to make a version of this function that can be exported
     FnSymbol* newFn = fn->copy();
     newFn->addFlag(FLAG_COMPILER_GENERATED);
+    newFn->addFlag(FLAG_LAST_RESORT);
```

Comment:
Let's add a comment to say why FLAG_LAST_RESORT is appropriate -- I'm supposing you want Chapel code to call the original and not the export wrapper?

Reply:
In the case where the arguments don't differ, we end up with a resolution conflict without this.  We do want to prioritize on calling the original version when we resolve the wrapper's body, too